CGPs/cgp-0060.md

```diff
@@ -256,6 +278,19 @@ The setup of the liquidity pool is based on the assumption that the cUSD/USDC pr
 
 The cUSD/USDCet pool will offer cUSD (and USDCet) at an incorrect price that is either higher (cUSD/USDCet < 1) or lower (cUSD/USDCet > 1) than the market price. The availability of expensive/cheap liquidity can have a price impact that is undesirable for cUSD as it is purely caused by a USDCet price deviation.
 
+ 
+## Market Interference
+Whenever arbitrageurs have matched cUSD/USDCe prices with the Stable Swap Pool, most users will trade with the reserve Stable Swap Pool. Due to the setup ([see Arbitrage and Dex Competition](#arbitrage-and-dex-competition) ) it has lower fee and slippage especially for larger trades. If DEX router include the Stable Swap Pool, this shift towards Stable Swap Pool will become even more likely.
+In other words DEXs have to compete with reserve liquidity. This is a dynamic which we would like to avoid in general. In the current situation  we believe that the benefit for the entire Celo ecosystem outweighs the risk of a competitive market intervention:
```

Comment:
```suggestion
In other words DEXs have to compete with reserve liquidity. This is a dynamic which we would like to avoid in general. In the current situation we believe that the benefit for the entire Celo ecosystem outweighs the risk of a competitive market intervention:
```